src/test/cloudWatchLogs/commands/searchLogGroup.test.ts

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as assert from 'assert'
+import {
+    SearchLogGroupWizard,
+    createLogGroupPrompter,
+    createFilterpatternPrompter,
+} from '../../../cloudWatchLogs/commands/searchLogGroup'
+import { AWSTreeNodeBase } from '../../../shared/treeview/nodes/awsTreeNodeBase'
+import { createQuickPickTester } from '../../shared/ui/testUtils'
+import { exposeEmitters, ExposeEmitters } from '../../../../src/test/shared/vscode/testUtils'
+import { InputBoxPrompter } from '../../../shared/ui/inputPrompter'
+import { createWizardTester } from '../../shared/wizards/wizardTestUtils'
+
+class FakeNode extends AWSTreeNodeBase {
+    public constructor(label: string) {
+        super(label)
+    }
+}
+
+describe('searchLogGroup', async function () {
+    const fakeLogNodes: AWSTreeNodeBase[] = []
+    let inputBox: ExposeEmitters<vscode.InputBox, 'onDidAccept' | 'onDidChangeValue' | 'onDidTriggerButton'>
+    let testPrompter: InputBoxPrompter
+
+    before(function () {
+        fakeLogNodes.push(new FakeNode('group-1'), new FakeNode('group-2'), new FakeNode('group-3'))
+        testPrompter = createFilterpatternPrompter()
+
+        inputBox = exposeEmitters(testPrompter.inputBox, ['onDidAccept', 'onDidChangeValue', 'onDidTriggerButton'])
+    })
+
+    it('Wizard accepts inputs', async function () {
+        const testWizard = createWizardTester(new SearchLogGroupWizard(fakeLogNodes))
+        const logGroupSelection = 'group-2'
+        testWizard.logGroup.applyInput(logGroupSelection)
+        testWizard.logGroup.assertValue(logGroupSelection)
+
+        const filterPatternSelection = 'this is filter text'
+        testWizard.filterPattern.applyInput(filterPatternSelection)
+        testWizard.filterPattern.assertValue(filterPatternSelection)
+    })
+
+    it('creates Log Group prompter from TreeNodes', async function () {
+        const prompter = createLogGroupPrompter(fakeLogNodes)
+        const tester = createQuickPickTester(prompter)
+        tester.assertItems(['group-1', 'group-2', 'group-3'])
+        const selection = 'group-2'
+        tester.acceptItem(selection)
+        tester.result(selection)
+    })
+
+    it('creates an valid InputBox', async function () {
+        assert.strictEqual(inputBox.title, 'Keyword Search')
+        assert.strictEqual(inputBox.placeholder, 'Enter text here')
+    })
+
+    it('filterPattern InputBox accepts input', async function () {
```

Comment:
It's ok to not test this as `createInputBox` + `InputBoxPrompter` is already tested elsewhere

Reply:
If I remove the other Wizard test in favor of a more flow focuses test, I should leave this specific tests in right? Looking at other examples, it seems like the convention is to only test the Wizard for flow as you mentioned, then test components individually.

Reply:
>  I should leave this specific tests in right? 

You can but what I'm saying is that the logic you're testing is already well-tested by other stuff. That is, what does this test do that `src/test/shared/ui/inputPrompter.test.ts` doesn't already?

I wouldn't worry about changing anything, just something to think about.

> it seems like the convention is to only test the Wizard for flow as you mentioned, then test components individually.

Yeah it tends to make things easier to read when the tests are split up.